### PR TITLE
Update copyright year to 2019

### PR DESCRIPTION
**Pull request by marcospereira**
## Purpose

Happy new year!

I did the replace inside Intellij, so it also made some other changes (like removing some whitespaces), and while inspecting them, I made some other small changes. But the main thing is just changing the year to have a green build.

Review with [?w=1](https://github.com/playframework/playframework/pull/8907/files?w=1). :-)

### CODE REVIEW

File: documentation/manual/working/commonGuide/filters/CspFilter.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ In particular, there are a number of subtle interactions with CSP that are unint
 
 ### False CSP Reports
 
-There can be a number of false positives produced from [browser extensions and plugins](https://jellyhive.com/activity/posts/2018/03/26/csp-implementations-are-broken/), and these can show as coming from `about:blank`.  It can take an extended period of time to resolve real issues and work out filters.  If you would rather configure a report-only policy externally, [Report URI](https://report-uri.com) is a hosted CSP service that will collect CSP reports and provide filters.
+There can be a number of false positives produced from [browser extensions and plugins](https://jellyhive.com/activity/posts/2019/03/26/csp-implementations-are-broken/), and these can show as coming from `about:blank`.  It can take an extended period of time to resolve real issues and work out filters.  If you would rather configure a report-only policy externally, [Report URI](https://report-uri.com) is a hosted CSP service that will collect CSP reports and provide filters.
```
Comment (dwijnand) [inline documentation/manual/working/commonGuide/filters/CspFilter.md]: Can't just replace all 2018 with 2019...

File: framework/src/play-integration-test/src/it/scala/play/it/http/AkkaResponseHeaderHandlingSpec.scala [hunk 2]
```diff
@@ -48,7 +48,7 @@ class AkkaResponseHeaderHandlingSpec extends PlaySpecification with AkkaHttpInte
     }
 
     "don't log a warning for Set-Cookie headers with negative ages" in {
-      val problemHeaderValue = "PLAY_FLASH=; Max-Age=-86400; Expires=Tue, 30 Jan 2018 06:29:53 GMT; Path=/; HTTPOnly"
+      val problemHeaderValue = "PLAY_FLASH=; Max-Age=-86400; Expires=Tue, 30 Jan 2019 06:29:53 GMT; Path=/; HTTPOnly"
```
Comment (dwijnand) [inline framework/src/play-integration-test/src/it/scala/play/it/http/AkkaResponseHeaderHandlingSpec.scala]: Could this be the cause of the test failure in CI?

**Comment by wsargent** [comment]
You can do this with IntelliJ IDEA's Copyright plugin and it'd be less hassle.

There's also https://github.com/sbt/sbt-header :-)

**Comment by marcospereira** [comment]
> 3 failures in the MultipartFormDataParserSpec integration test:

Missing whitespace. See https://github.com/playframework/playframework/pull/8907/commits/0b982a462f4826f11fa086db22672a142b259414.

**Comment by marcospereira** [comment]
Closing in favor of #8908.

**Comment by dwijnand** [comment]
> > 3 failures in the MultipartFormDataParserSpec integration test:
> 
> Missing whitespace. See [0b982a4](https://github.com/playframework/playframework/commit/0b982a462f4826f11fa086db22672a142b259414).

Ouch!  That's a pain.

**Pull request closed without merging**